site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -186,17 +186,17 @@ This will produce a compiled version of your code in the `dist` folder, along wi
 
 It is worth updating the `blockprotocol` object in `package.json` to include your own `icon`, `image`, and `examples` for your block. These will automatically be included in the `block-metadata.json` produced after running `yarn build` or `npm run build`.
 
-You now have a block package that you can provide to apps to use, by **publishing it on the Block Hub**.
+You now have a block package that you can provide to apps to use, by **publishing it on the Hub**.
```

Comment:
We alternate between _on the Hub_ and _to the Hub_. @nonparibus shall we use _on the Hub_ everywhere?

Reply:
@nonparibus https://github.com/blockprotocol/blockprotocol/pull/612/commits/57ce5c7d73076e1e67001816d1fc28015a7e37b3